packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.types.ts

```diff
@@ -1,12 +1,50 @@
-import { ITheme } from '../../Styling';
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
 
+/**
+ * {@docCategory VerticalDivider}
+ * Props for the Vertical Divider
+ */
 export interface IVerticalDividerProps {
   /**
+   * @deprecated Use styles instead.
    * Optional function to generate the class names for the divider for custom styling
    */
   getClassNames?: (theme: ITheme) => IVerticalDividerClassNames;
+  /**
+   * The theme that should be used to render the vertical divider.
+   */
+  theme?: ITheme;
+  /**
+   * Optional override stylings that will get merged with the dividers styles.
+   */
+  styles?: IStyleFunctionOrObject<IVerticalDividerPropsStyles, IVerticalDividerStyles>;
+  /**
+   * className that will be placed on the divider wrapper div
+   */
+  className?: string;
+}
+
+/**
+ * {@docCategory VerticalDivider}
+ * Props that will get passed to the styling function to style the Vertical Divider
+ */
+export type IVerticalDividerPropsStyles = Pick<IVerticalDividerProps, 'theme' | 'getClassNames' | 'className'>;
+
+/**
+ * {@docCategory VerticalDivider}
+ * Style interface that defines the different areas that styles can be customized on the Vertical Divider
+ */
+export interface IVerticalDividerStyles {
+  wrapper: IStyle;
```

Comment:
nit: Document these areas. We wanna have as detailed as possible documentation even if it's kinda obvious by the name we still wanna set an example by documenting everything.